pkgs/tools/filesystems/rar2fs/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, fetchurl
+, fuse
+}:
+
+with stdenv.lib;
+
+let
+
+  version = "1.28.0";
+
+  unrar-src = fetchurl {
+    url = "https://www.rarlab.com/rar/unrarsrc-5.8.5.tar.gz";
+    sha256 = "0abvz6vv8kr416fphysfbwgxc6hyf1bpnd0aczfv7j3vc8x949d7";
+  };
```

Comment:
Can't you depend on the existing `unrar` package (`pkgs/tools/archivers/unrar/default.nix`) instead of fetching and building it here?

Let me know if it causes problems (e.g. if the build scripts don't find the system-wide library).

Reply:
I tried that but the build scripts wants the source while compiling rar2fs. Will try again :)

Reply:
If the source is needed it should work with something like "--with-unrar=${unrar.src}" (if the store path is already extracted). I also had a look at [configure.ac](https://github.com/hasse69/rar2fs/blob/b6d1d42952d8ec7153bc1c859de88b288cf6b5d6/configure.ac#L209) and `--disable-static-unrar` probably makes sense as well as the default is to link statically (that might be why the source-code is required).

Reply:
Also here I get
```
these derivations will be built:
  /nix/store/h0v6fvccw445zv7n4adbjl48f8fk7bwd-rar2fs-1.28.0.drv
error: some outputs of '/nix/store/h0v6fvccw445zv7n4adbjl48f8fk7bwd-rar2fs-1.28.0.drv' are not valid, so checking is not possible
```
when introducing
```
  configureFlags = [
    "--disable-static-unrar"
    "--with-unrar=${unrar.src}"
  ];
```
incombination `buildInputs = [ fuse unrar ];` and `{ unrar }` in the arguments in the top of the nix-file.